### PR TITLE
Support for vertx-web

### PR DESCRIPTION
**Pull request by IshikaDawda**
Introduce 3 modules for vertx support:
- vertx-web-3.2.0: ranging from version 3.2.0 to 3.5.1(excluded)
- vertx-web-3.5.1: ranging from version 3.5.1 to 3.8.3(excluded)
- vertx-web-3.8.3: ranging from version 3.8.3 to latest

Events are dropping, as they are classified as internal events, due presence of traces `com.nr.vertx.instrumentation.VertxUtil$1` in the stacktrace. 
Need to filter out `com.nr.vertx.instrumentation.VertxUtil$1` from the stacktrace of events.

Changelogs:
=========
- [PR-241](https://github.com/newrelic/csec-java-agent/pull/241) Vert.x-Web Support : The security agent now also supports Vert.x-Web Version 3.2.0 and above. [NR-254180](https://new-relic.atlassian.net/browse/NR-254180), [NR-254181](https://new-relic.atlassian.net/browse/NR-254181), [NR-254182](https://new-relic.atlassian.net/browse/NR-254182)
- [PR-245](https://github.com/newrelic/csec-java-agent/pull/245) Vert.x-Core Support : The security agent now also supports Vert.x-Core Version 3.3.0 and above. [NR-254146](https://new-relic.atlassian.net/browse/NR-254146), [NR-254156](https://new-relic.atlassian.net/browse/NR-254156)
- [PR-242](https://github.com/newrelic/csec-java-agent/pull/242) Fix for User Class detection in Play Framework [NR-264101](https://new-relic.atlassian.net/browse/NR-264101)

[NR-254180]: https://new-relic.atlassian.net/browse/NR-254180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NR-254181]: https://new-relic.atlassian.net/browse/NR-254181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NR-254182]: https://new-relic.atlassian.net/browse/NR-254182?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NR-254146]: https://new-relic.atlassian.net/browse/NR-254146?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by IshikaDawda** [comment]
Please merge this Pull Request after merging #245 and #242.

**Review by lovesh-ap — APPROVED** [review]

**Pull request merged**
